test/metabase/query_processor/streaming/xlsx_test.clj

```diff
@@ -110,126 +109,137 @@
-                                                        :type/Price)))
-        (is (= "[$EUR] #,##0.00"         (format-string {::mb.viz/currency-in-header false,
-                                                         ::mb.viz/currency "EUR",
-                                                         ::mb.viz/currency-style "code"}
-                                                        :type/Price)))
-        (is (= "#,##0.00\" euros\""      (format-string {::mb.viz/currency-in-header false,
-                                                         ::mb.viz/currency "EUR",
-                                                         ::mb.viz/currency-style "name"}
-                                                        :type/Price))))
-
-      (testing "Currency not included for non-currency semantic types"
-        (is (= "#,##0.00" (format-string {::mb.viz/currency-in-header false} :type/Quantity))))
-
-      (testing "Formatting options are ignored if currency-in-header is true or absent (defaults to true)"
-        (is (= "#,##0.00" (format-string {::mb.viz/currency-style "symbol"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::mb.viz/currency-style "name"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::mb.viz/currency-style "code"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::mb.viz/currency "USD"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::mb.viz/currency "EUR"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::currency-in-header true, ::mb.viz/currency-style "symbol"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::currency-in-header true, ::mb.viz/currency-style "name"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::currency-in-header true, ::mb.viz/currency-style "code"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::currency-in-header true, ::mb.viz/currency "USD"} :type/Price)))
-        (is (= "#,##0.00" (format-string {::currency-in-header true, ::mb.viz/currency "EUR"} :type/Price))))
-
-      (testing "Global localization settings are incorporated with lower precedence than column format settings"
-        (mt/with-temporary-setting-values [custom-formatting {:type/Currency {:currency "EUR",
-                                                                              :currency_in_header false,
-                                                                              :currency_style "code"}}]
-          (is (= "[$EUR] #,##0.00" (format-string {} :type/Price)))
-          (is (= "[$CAD] #,##0.00" (format-string {::mb.viz/currency "CAD"} :type/Price)))
-          (is (= "[$€]#,##0.00"    (format-string {::mb.viz/currency-style "symbol"} :type/Price)))
-          (is (= "#,##0.00"        (format-string {::mb.viz/currency-in-header true} :type/Price))))))
+      (let [price-col {:semantic_type :type/Price, :effective_type :type/Float}]
```

Comment:
Almost all the changes are due to the addition of this binding, which each test passes into `format-string` as the last arg